editor.planx.uk/src/components/Feedback/index.tsx

```diff
@@ -364,7 +375,11 @@ const Feedback: React.FC = () => {
     }
   }
 
-  return <Feedback />;
+  return (
+    <div ref={feedbackComponentRef}>
```

Comment:
nit: Throughout the codebase we tend to use MUI components, could we change this `div` to a `Box` for consistency?